apps/sms/style/sms.css

```diff
@@ -833,7 +833,7 @@ Do not remove.
 /* Adjust the name line-height to prevent bottom of chars been cut */
```

Comment:
wondering why this line-height adjustment is not done in lists.css along with the padding change ?